deploy.sh

```diff
@@ -1,9 +1,12 @@
 #!/usr/bin/env bash
+echo "Running on $TRAVIS_OS_NAME"
+
 if [[ "$TRAVIS_OS_NAME" == "osx" ]]
 then
   cd ios && fastlane release
 elif [[ "$TRAVIS_OS_NAME" == "linux" ]]
 then
+      export SUPPLY_VERSION_CODE=211
```

Comment:
Is it really necessary?

Reply:
Yes

Reply:
Yes

Reply:
1. Why?
2. Cannot us get it from the gradle file?

Reply:
It wouldn't work for Fastlane. This fixes the issue.